espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/w3c/adapter/DummyW3CActionAdapter.java

```diff
@@ -85,7 +88,7 @@ public double getPointerMoveDurationMargin(PointerInputState pointerInputState)
     }
 
     public void pointerMove(String sourceId, PointerType pointerType,
-                            long currentX, long currentY, long x, long y,
+                            float currentX, float currentY, float x, float y,
```

Comment:
This is not very consistent. Sometimes `float` is used and in other places `Float`. Also FYI: JSON parser transforms non-integer numbers to Double by default, so we do perform an unnecessary type conversion here

Reply:
The usage of primitives vs. objects is admittedly arbitrary. When I was working on the Actions api I often made the choice based on whichever passed

Reply:
I would just prefer the stuff to be consistent everywhere